src/main.cc

```diff
@@ -47,7 +47,7 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       unsigned short v1, v2, v3, v4;
       if (!parse_version_string(argv[++i], &v1, &v2, &v3, &v4))
-        return print_error("Unable to parse version string");
+        return print_error("Unable to parse version string for FileVersion");
```

Comment:
I guess `sprintf` could be used but that adds more lines. I am not aware of a standard function that takes in varargs and returns a newly allocated `char *`.